dev.exs

```diff
@@ -1,9 +1,17 @@
 # iex -S mix dev
 
 Logger.configure(level: :debug)
-
+defmodule Surface.Catalogue.ErrorView do
+  use Phoenix.View,
+    root: "lib/surface/catalogue/templates",
+    namespace: Surface.Catalogue
+  def template_not_found(template, _assigns) do
+    Phoenix.Controller.status_message_from_template(template)
+  end
+end
```

Comment:
If you update the catalogue to `v0.0.2` or `v0.0.3`, you don't need this anymore.